extensions/kafka-connect/src/main/java/com/hazelcast/jet/kafka/connect/impl/SourceOffsetStorageReader.java

```diff
@@ -40,7 +40,7 @@ public <V> Map<String, Object> offset(Map<String, V> partition) {
     public <V> Map<Map<String, V>, Map<String, Object>> offsets(Collection<Map<String, V>> partitions) {
         Map<Map<String, V>, Map<String, Object>> map = new HashMap<>();
         for (Map<String, V> partition : partitions) {
-            Map<String, Object> offset = (Map<String, Object>) partitionsToOffset.get(partition);
+            Map<String, Object> offset = (Map<String, Object>) state.getPartitionsToOffset().get(partition);
```

Comment:
```suggestion
            Map<String, Object> offset = (Map<String, Object>) state.getPartitionOffset(partition);
```
getPartitionOffset is a new method  for readability